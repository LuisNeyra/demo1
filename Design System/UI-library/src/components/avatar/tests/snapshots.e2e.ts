import { snapIt } from '../../../test';

const component = 'avatar';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('nothing if no attributes are set', '<blaze-avatar></blaze-avatar>');
    snap('text inside', '<blaze-avatar text="GP"></blaze-avatar>');
    snap('correctly sized circles', '<blaze-avatar size="super" text="GP"></blaze-avatar>');
    snap('an image', '<blaze-avatar src="my-test-image.png"></blaze-avatar>');
    snap('2 images', '<blaze-avatar src="my-test-image.png" src2="my-second-test-image.png"></blaze-avatar>');
    snap(
      'images, text and sizes it correctly',
      '<blaze-avatar src="my-test-image.png" src2="my-second-test-image.png" size="super" text="GP"></blaze-avatar>'
    );
  });
});
