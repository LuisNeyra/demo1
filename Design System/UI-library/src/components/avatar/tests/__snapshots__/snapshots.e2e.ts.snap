// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`avatar renders 2 images 1`] = `"<blaze-avatar src=\\"my-test-image.png\\" src2=\\"my-second-test-image.png\\" class=\\"hydrated\\"><div class=\\"c-avatar\\"><img class=\\"c-avatar__img\\" src=\\"my-test-image.png\\"></div></blaze-avatar>"`;

exports[`avatar renders an image 1`] = `"<blaze-avatar src=\\"my-test-image.png\\" class=\\"hydrated\\"><div class=\\"c-avatar\\"><img class=\\"c-avatar__img\\" src=\\"my-test-image.png\\"></div></blaze-avatar>"`;

exports[`avatar renders correctly sized circles 1`] = `"<blaze-avatar size=\\"super\\" text=\\"GP\\" class=\\"hydrated\\"><div class=\\"c-avatar u-super\\" data-text=\\"GP\\"></div></blaze-avatar>"`;

exports[`avatar renders images, text and sizes it correctly 1`] = `"<blaze-avatar src=\\"my-test-image.png\\" src2=\\"my-second-test-image.png\\" size=\\"super\\" text=\\"GP\\" class=\\"hydrated\\"><div class=\\"c-avatar u-super\\" data-text=\\"GP\\"><img class=\\"c-avatar__img\\" src=\\"my-test-image.png\\"></div></blaze-avatar>"`;

exports[`avatar renders nothing if no attributes are set 1`] = `"<blaze-avatar class=\\"hydrated\\"></blaze-avatar>"`;

exports[`avatar renders text inside 1`] = `"<blaze-avatar text=\\"GP\\" class=\\"hydrated\\"><div class=\\"c-avatar\\" data-text=\\"GP\\"></div></blaze-avatar>"`;
